src/NinjaTools.FluentMockServer/Xunit/MockServerContainer.cs

```diff
@@ -14,15 +16,19 @@
     /// </summary>
     public class MockServerContainer : IDisposable
     {
+        public const string Prefix = "NinjaTools-FluentMockServer";
+        public const string Suffix = "netcore-xunit-background-runner";
+        public static string ContainerName = $"{Prefix}-{Suffix}";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'ContainerName' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)